### ISSUE TITLE
Add a mapping from persistentIdentity to identity of the latest version

### ISSUE THREAD

**Issue by cjmyers** [SynBioHub/synbiohub#157]
This predicate could be used to determine what is the latest version and also be able to fetch by persistentIdentity.  It would need to be updated every time a new version is added.

**Comment by cjmyers** [comment]
Instead lets do a redirect to latest version.

**Comment by cjmyers** [comment]
This is done now, but there are a few issues:

1) SynBioHub is now strict about the URI path, namely:
	<repoURL>/public/collectionId/displayId/optionalVersion
	<repoURL>/user/userId/collectionId/displayId/optionalVersion

In other words, there can not be any extra paths.  This means the igem registry collections needed to be changed to remove the extra paths.  Now, the displayId is the category name with “/“ replaced by “_”.  This means before checking out, you must first checkout and rerun the igem script, remove your current igem collection, and upload the new one.

2) The semver package I used requires versions to include three digits separated by periods: 
	MAJOR.MINOR.PATCH

In SBOL, for some reason, we decided not to require MINOR and PATCH and assume to be 0 if missing.  Either we misinterpreted semantic versioning as not requiring the minor/patch versions or found some other definition somewhere that allowed this.  To address this, I have to add “.0” on manually to versions when using the semver package for comparisons or it throws an exception saying invalid version.  It is a bit of a hack, but did not see any other way to get around this problem short of writing my own semver package.

3) The linking in sboljs does not link by persistentIdentity.  This means if an sbol file has references that use the persistentIdentity, then it will not create a link to the object.  The problem is that when you try to lookup information about the object, it will not be present.  One obvious place this affects is in rendering in VisBol, since it is unable to find the information about the ComponentDefinition being referenced by the Component.  Likely, the fix is to update sboljs to allow for persistentIdentity linking.
